twigextensions/ElixirTwigExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Craft;
+
+use Twig_Extension;
+use Twig_Filter_Method;
+use Twig_Function_Method;
```

Comment:
Also appear to be deprecated in favor of `Twig_SimpleFunction`
http://twig.sensiolabs.org/doc/deprecated.html#functions